src/OrbitGl/CaptureWindow.cpp

```diff
@@ -121,8 +121,9 @@ void CaptureWindow::MouseMoved(int x, int y, bool left, bool right, bool middle)
   if (left && !picking_manager_.IsDragging() && !app_->IsCapturing()) {
     Vec2i mouse_click_screen = viewport_.WorldToScreen(mouse_click_pos_world_);
     Vec2 mouse_pos_world = viewport_.ScreenToWorld({x, y});
-    time_graph_->SetVerticalScrollingOffset(timegraph_click_scrolling_offset_ +
-                                            mouse_click_pos_world_[1] - mouse_pos_world[1]);
+    float new_vertical_offset =
+        timegraph_click_scrolling_offset_ + mouse_click_pos_world_[1] - mouse_pos_world[1];
+    time_graph_->SetAndClampYPos(-new_vertical_offset);
```

Comment:
Could you add a comment explaining why this is negative?

Reply:
No longer needed after my update